requirements/core.txt

```diff
@@ -1,14 +1,9 @@
 numpy==1.19.0
 scipy==1.5.0
 statsmodels==0.11.1
-pmdarima==1.6.1
 matplotlib==3.2.2
-fbprophet==0.6
 pandas==1.1.0
-torch==1.5.1
-tensorboard==2.2.2
 ipython==7.15.0
 tqdm==4.46.1
 holidays==0.10.2
-plotly==4.9.0
-ipywidgets==7.5.1
```

Comment:
Do we need to run some checks to see whether removing `plotly` and ìpywidgets` will not result in any unexpected behavior? I'm just thinking that there must have been a reason for why we added those..

Reply:
I think they are old remnants. I'll trust the unit tests (and mostly the notebook examples tests) on that one :)